hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/MemberPartitionStateImpl.java

```diff
@@ -37,6 +37,7 @@
 
     List<Integer> partitions = new ArrayList<Integer>(DEFAULT_PARTITION_COUNT);
     boolean memberStateSafe;
+    @Deprecated
```

Comment:
Why is it deprecated instead of removed?

Reply:
I see that it is removed now.

Reply:
It was an overlook. I've just pushed be0c0bd1e6d7d4a207c55d92793ca8f8a0a6e601 which fixes it.